src/__mocks__/Session.mock.ts

```diff
@@ -52,14 +52,26 @@ class Session extends BaseSession {
     return true;
   }
 
+  hasVideoTracks(mediaStream: MediaStream): boolean {
```

Comment:
don't create method, use function (this is not used)

Reply:
done